sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+
+Throughout this release, sBTC will have an unchangeable and distinct signer set that will not explicitly be part of Stacks consensus. As a result, this release can activate without a hard fork. Each unique signer receives exactly one vote. Given there are 15 unique signers that comprise the system, it would require at least 11 out of 15 signatures for an sBTC operation to be fulfilled.
+
+While up to 30% of the signers can be offline without a user impact on the functioning of the protocol, it becomes more critical for the rest of the signers to approve sBTC operations because operations necessarily still need to meet 70% of the original signing power. If more than 30% of signers become unavailable, no sBTC operations will be approved because it will be impossible to get 70% approval when less than 70% are online. An operation that isn’t approved will become spendable by the user after a predetermined timeout has elapsed, measured in Bitcoin blocks.
+
+### sBTC Signer Responsibilities
+
+Overview of tasks the sBTC signers carry out:
+    - They react to requests to deposit BTC.
+    - They react to requests to withdraw BTC.
+    - They stay online often enough that deposits and withdrawals happen in a timely manner.
+    - They have to keep their signer hosts secure, so their keys don't get stolen.
+    - They have to proactively re-key every so often, which means moving the BTC to a new UTXO.
+- Signers must consolidate BTC as it is deposited.
+    - The process for UTXO consolidation of the BTC peg wallet is described in this [issue](https://github.com/stacks-network/sbtc/issues/52).
+- Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions.
+    - They must ensure that the transaction fee is paid for (e.g., they deduct it from the user, and they set a minimum sBTC withdrawal amount).
+    - The process for handling transaction fee estimates is described in this [issue](https://github.com/stacks-network/sbtc/pull/186).
+- Signers must coordinate to set and advertise the fee parameters of the system.
+    - They must decide a minimum sBTC peg-out.
+    - They must decide an STX transaction fee for minting the sBTC. This fee is paid by the user and can be sponsored by a 3rd party, which is described in the “Auxiliary Features” section of this document.
+
+### sBTC Signer Eligibility Criteria
+
+Signers will run the sBTC binary in addition to the core Stacks signer software and must meet certain criteria in order to facilitate the reliable functioning of the sBTC protocol at all times.
```

Comment:
Why is it required to run a core Stacks signer?

Reply:
I'm also confused by this.  I can see a reason for sBTC signers to run Stacks nodes, but why signers?  Are sBTC signers required to first be block signers in Nakamoto?

Reply:
I read this as a proposed requirement for an initial sBTC signer set is that they also run a stacks node signer. 
there's no technical reason this needs to be true, but for the eligibility of the first set of chosen signers - this is an imposed requirement. is that what you're suggesting @andrerserrano ?

Reply:
Adding on here - i think the language here needs a little update to remove pronouns. 
ex: 
`They must decide a minimum sBTC peg-out.` -> `A minimum sBTC peg-out must be decided`

keeping these responsibilities agnostic reads a bit better than adding a pronoun "they" (etc) to each item in the list.

Reply:
addressed the pronouns in the list. 
@aldur how do you feel about the explantation around the sbtc signer also running a stacks signer?

Reply:
Yes, running a Stacks signer is not a technical requirement, but it shows that they have experience with the codebase and provides more data to evaluate them by.

Reply:
I had to go look at the sBTC signer code to determine this, but I think it's relevant to call out that the sBTC signer is a Stacks event observer, meaning that the experience of running a Stacks node signer directly translates to running an sBTC signer (this isn't obvious from the text).

They don't have to be a Stacker on mainnet, right?  It suffices that they run a Stacks signer as part of mock-signing or on testnet as well, right?

Reply:
That's correct. Good call out, I will add that text to the SIP.